ftdetect/go.vim

```diff
@@ -0,0 +1,6 @@
+" For now, rely on the .go detection, syntax, indent, etc. provided in recent
+" Vim distributions.
+
+" By default, vim associates .mod files with filetypes lprolog or modsim3.
+" Override these rather than using setfiletype.
+autocmd BufNewFile,BufRead go.mod setlocal filetype=gomod
```

Comment:
This doesn't feel quite right here. Is there a reason it needs to be here, or could it live in `plugin/govim.vim`?

Reply:
AFAIK putting this in the `ftdetect` directory is the standard layout for Vim plugins. It also has special semantics: vim automatically creates an augroup for the commands in the ftdetect file.